data/scripts/obtain_item.inc

```diff
@@ -103,6 +103,9 @@ Std_FindItem::
 	lock
 	faceplayer
 	waitse
+#if RZ_ENABLE == TRUE
+	callnative FindItemRandomize_NativeCall
+#endif
```

Comment:
A bit of a global comment, but we use `.if` and `.endif` in .inc files to check macros